js/index.js

```diff
@@ -1,491 +0,0 @@
-          if(packageA.speed === 0){  
-            if (packagesOverlap(packageA, packageB)) {
-            packageA.speed = 0;
-            packageB.speed = 0;
-          }
-        }
-
-      
-      }
-    }
-
-    //set game over condition
-   if(package.speed === 0 && package.x <= 20 ){
-    gameOver();
-    console.log("game over")
-   }
-      
-
-    })
-
-
-    //draw loaded packages
-    loadedPackage.forEach((package)=>{
-      
-      stroke(161,115,76);
-      strokeWeight(1);
-      fill(184,136,91)
-      rect(package[0].x, package[0].y, package[0].w, package[0].h) 
-      //tape
-      stroke(161,115,76);
-      strokeWeight(10);
-      drawTape(package[0].x + 5,package[0].y + 5 ,package[0].w - 10,package[0].h - 10)
-     
-      //sticker 
-      stroke(229,226,228)
-      fill(229,226,228)
-      rect(package[0].x + 15,package[0].y + 20, 30, 15)
-
-      //text in sticker
-      strokeWeight(0)
-      fill(0);
-      textSize(12);
-      text(package[0].text, package[0].x + 30, package[0].y + 30);})
-
-
-      //typedText
-      strokeWeight(10)
-      stroke(213,0,41)
-      fill(254,204,0);
-      textSize(100);
-      text(typedText, 150,height - 200)
-      // ('color', color(213,0,41));
-      // button.style('background-color', color(254,204,0));
-
-      if(leftH === 250 && leftW === 0){
-        level = level * 0.8
-        leftW = insideW
-        leftH = 0
-        loadedPackage = []
-        messageLevelUp = true;
-      }
-
-      //level up message  
-      if(messageLevelUp === true){
-        strokeWeight(0)
-        fill(213,0,41);
-        textSize(50);
-        text("You rock it! Let's work faster!", (width / 2) - (insideW / 2)+300, height - 400);
-        setTimeout(() => {
-          messageLevelUp = false
-        }, 1500);}
-
-
-      //Warntext
-      if(messageVisible === true){
-        strokeWeight(0)
-        fill(213,0,41);
-        textSize(50);
-        text("This package is too big", (width / 2) - (insideW / 2)+300, height - 400);
-
-        setTimeout(() => {
-          messageVisible = false
-        }, 1500);
-      }
-      
-
-        restartButton.hide()
-
-    }else if(gameState === 2){
-      noStroke()
-      fill(color(232,232,232,100))
-      rect(width / 2 - 325, height / 2 - 250, 650 , 400)
-
-      // color', color(213,0,41));
-      // background-color', color(254,204,0));
-      fill(213,0,41);
-      textAlign(CENTER);
-      textSize(32);
-      text('Game Over',  width / 2 - 280, height / 2 - 70 ,600, 400);
-
-      restartButton.show()
-
-    }
-
-    
-  
-  }
-
-function startGame() {
-  gameState = 1;
-  button.hide();
-  song.play()
-  song.loop()
-}
-
-function restartGame() {
-  gameState = 1;
-  restartButton.hide();
-  button.hide()
-  randomARR = []
-  loadedPackage = []
-  typedText =""
-  score = 0
-  song.play()
-  song.loop()
-  level = 2500;
-}
-
-function gameOver() {
-  gameState = 2;
-  song.stop()
-  gameOverSound.play()  
-}
```

Comment:
It's really nice when code is structured in such a way that parts with things in common are together like this, makes your code very easy to navigate